examples/oidc/css-api/nextjs/src/pages/api/integrations/[integrationID]/[environment]/users/[username]/roles/index.js

```diff
@@ -0,0 +1,29 @@
+import { fetchCssApiCredentials, apiErrorHandler, handleStatusError } from '@/utils'
+
+async function handler(req, res) {
+    const { integrationID, environment, username } = req.query
+    const accessToken = await fetchCssApiCredentials();
+
+    // Add a role to the provided user.
+    // See https://api.loginproxy.gov.bc.ca/openapi/swagger#/Role-Mapping for details.
+    if (req.method === 'POST') {
+        const roles = await fetch(`${process.env.SSO_API_URL}/integrations/${integrationID}/${environment}/users/${username}/roles`, {
+            headers: {
+                authorization: `Bearer ${accessToken}`,
+                'Content-Type': 'application/json',
+                'Accept': 'application/json'
+            },
+            body: req.body,
+            method: "POST"
+        })
+        if (roles.status >= 400) {
```

Comment:
what about 300 errors?  Not sure if our api even throws those

Reply:
I went for 400+ as errors, in case of like a 304 not modified. Looking at our app though we stick to just 200, 400, and 500's so don't think there ever would be a 300 level response